src/Administration/Resources/app/administration/src/module/sw-property/snippet/de-DE.json

```
@@ -31,6 +31,7 @@
       "labelName": "Name",
       "placeholderName": "Namen eingeben ...",
       "labelFilterable": "Im Produktfilter von Produktlisten anzeigen",
+      "labelVisibleOnDetail": "Anzeige auf der Produktdetailseite",
```

```suggestion
      "labelVisibleOnDetail": "Auf der Produktdetailseite anzeigen",
```
... to use the same phrase as `labelFilterable`